src/home/home.html

```diff
@@ -77,7 +77,7 @@
         </div>
 
         <div class="heroColumn">
-          <iframe class="introVideo" src="https://www.youtube.com/embed/NJBQj_cZlP8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+          <iframe class="introVideo" src="https://www.youtube.com/embed/FhE4kWEiCJM" title="Prime Deals - Web3 partnerships made easy" frameborder="0" allowfullscreen></iframe>
```

Comment:
Why were these attributes removed?
`allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture`

Reply:
because all of them seem to
1. be irrelevant.
2. not having any effect on the outcome. (Or maybe on mobile? I didn't check).

Reply:
> I don’t know where those args came from, but I didn’t see that in google documentation…
https://developers.google.com/youtube/iframe_api_reference